### DISCUSSION TITLE
Some elements couldn’t be recognized in Appium Inspector after Appium, Xcuitest, Xcode and iOS upgrade

### DISCUSSION THREAD

**Discussion by MilaLynx** [appium/.github#10] [Q&A]
Made the following upgrades:

    real device iOS version from 15.6 to 18
    Xcode from 14.3.1 to 16
    Appium from 2.0.1 to 2.11.5
    xcuitest from 4.12.0 to 7.27.0
    After those upgrades elements of app that are “native” (popups and so on) become unaccessible from Appium Inspector and for automated tests.

Capabilities I use:

{
  "appium:deviceName": "iPhone 12",
  "appium:udid": "00008101-00127CAA3AE0001E",
  "appium:platformVersion": "18.0",
  "appium:automationName": "XCUITest",
  "appium:bundleId": "com.onoffapp.OnOffAppProduction",
  "appium:useNewWDA": "true",
  "platformName": "iOS",
  "showXcodeLog": true
}

Before upgrade the Appium Inspector contents looked like this and Call.../Cancel buttons were accessible:
![Screenshot 2024-10-01 at 14 28 43](https://github.com/user-attachments/assets/5b9c8296-bb2f-486f-aa1a-2be79970f9ae)

After upgrade it looks like this and these buttons cannot be recognized as elements:
<img width="1090" alt="Screenshot 2024-10-01 at 13 25 27" src="https://github.com/user-attachments/assets/607f1016-6bc3-4238-bc1e-30964d5b97f1">

It seems to me, that somewhere is a setting or something like that, which has to be switched on, but I cannot get it, where it is.

**Comment by KazuCocoa** [comment]
https://appium.github.io/appium-xcuitest-driver/latest/guides/troubleshooting/#interact-with-dialogs-managed-by-comapplespringboard might help
